cmd/sonrd/utils/keychain.go

```diff
@@ -0,0 +1,123 @@
+package utils
+
+import (
+	"bytes"
+	"encoding/gob"
+
+	"github.com/pkg/errors"
+	"github.com/sonr-io/keyring"
+	"github.com/sonr-io/sonr/pkg/crypto/mpc"
+	"github.com/sonr-io/sonr/pkg/fs"
+	mt "github.com/sonr-io/sonr/third_party/types/motor"
+)
+
+const K_SERVICE_NAME = "sonr-dev"
```

Comment:
?

Reply:
Keychain service name - will be easier to query across motor nodes